group-coordinator/src/main/java/org/apache/kafka/coordinator/group/share/ShareGroupMember.java

```diff
@@ -236,4 +240,53 @@ public String toString() {
             ", assignedPartitions=" + assignedPartitions +
             ')';
   }
+
+  private static String lookupTopicNameById(
+      Uuid topicId,
+      TopicsImage topicsImage
+  ) {
+    TopicImage topicImage = topicsImage.getTopic(topicId);
+    if (topicImage != null) {
+      return topicImage.name();
+    } else {
+      return null;
+    }
+  }
```

Comment:
Doesn't seems like have any dependency on `ShareGroupMember` so shouldn't we move the method to some common class, may be in `Utils` and access same rather having 2 copies in Share and Consumer Group Member?